extensions/amp-story/1.0/amp-story.js

```diff
@@ -1064,10 +1064,57 @@ export class AmpStory extends AMP.BaseElement {
       });
     }
 
+    this.buildScreenReaderBackButton_();
+
     // Will resolve when all pages are built.
     return storyLayoutPromise;
   }
 
+  /**
+   * Creates a focusable button for screen readers to navigate back.
+   * @private
+   */
+  buildScreenReaderBackButton_() {
+    const backButton = document.createElement('button');
+    this.element.appendChild(backButton);
+    backButton.classList.add('i-amphtml-story-screen-reader-back-button');
+    backButton.setAttribute('aria-label', 'Previous page');
+
+    backButton.addEventListener('click', () => {
+      this.previous_();
+    });
+
+    this.storeService_.dispatch(
+      Action.SET_ADVANCEMENT_MODE,
+      AdvancementMode.MANUAL_ADVANCE
+    );
+
+    hideOnFirstPage(
+      Number(this.storeService_.get(StateProperty.CURRENT_PAGE_INDEX))
+    );
+
+    this.storeService_.subscribe(
+      StateProperty.CURRENT_PAGE_INDEX,
+      (pageIndex) => hideOnFirstPage(pageIndex)
+    );
+
+    /**
+     * Append class to hide button if on first page.
+     * @param {number} pageIndex
+     * @return {void}
+     * @private
+     */
+    function hideOnFirstPage(pageIndex) {
+      pageIndex === 0
+        ? backButton.classList.add(
+            'i-amphtml-story-screen-reader-back-button-hidden'
+          )
+        : backButton.classList.remove(
+            'i-amphtml-story-screen-reader-back-button-hidden'
+          );
```

Comment:
Can you wrap this into a `this.mutateElement()` statement?

Nit: you can use the `classList.toggle(string, boolean)` method